test_images/Docker.centos7.aarch64

```diff
@@ -0,0 +1,3 @@
+FROM arm64v8/centos:7
```

Comment:
Do you want to make it more procedural?
```bash
TEST_DOCKER_IMAGES="arm64v8/centos:7 arm64v8/ubuntu:16.04"
for TEST_IMAGE in $TEST_DOCKER_IMAGES; do
    echo "FROM $TEST_IMAGE" > DOCKERFILE
    echo "ADD qemu/qemu-${ARCH}-static /usr/bin/qemu-aarch64-static" >> DOCKERFILE
    echo "ENV ARCH=${ARCH}" >> DOCKERFILE
done
```

Reply:
I thought about it but having custom Dockerfiles allows us to be more flexible in the future and also make it easier for people to directly see what is being tested just by looking at the name of the files in the `test_image` folder.

That being said I don't have strong feelings about it so let me know what do you prefer.

Reply:
yeah, i also hate having files create files that create files that create files... all makes sense